src/membrane-keypad-element.ts

```diff
@@ -11,23 +20,33 @@ export class MembraneKeypadElement extends LitElement {
   @property() threeColumns = false;
 
   private pressedKeys = new Set<string>();
+  private shiftPressed = false;
 
   renderKey(text: string, x: number, y: number) {
     const keyClass = isNumeric(text) ? 'blue-key' : 'red-key';
+    let keyId = `key${text}`;
+    if (text === '*') {
+      keyId = 'keyAsterisk';
+    } else if (text === '#') {
+      keyId = 'keyHash';
+    }
 
     return svg`<g
       transform="translate(${x} ${y})"
       tabindex="0"
       class=${keyClass}
+      id=${keyId}
       @blur=${(e: FocusEvent) => {
         this.up(text, e);
       }}
       @mousedown=${() => this.down(text)}
       @mouseup=${() => this.up(text)}
       @touchstart=${() => this.down(text)}
       @touchend=${() => this.up(text)}
-      @keydown=${(e: KeyboardEvent) => e.keyCode === SPACE_KEY && this.down(text, e)}
-      @keyup=${(e: KeyboardEvent) => e.keyCode === SPACE_KEY && this.up(text, e)}
+      @keydown=${(e: KeyboardEvent) =>
+        (e.keyCode === SPACE_KEY || e.code === 'Space') && this.down(text, e)}
```

Comment:
I used a cutom keyboardEvent, and since keyCode is deprecated it can't be defined, so I used code instead. But now I removed the custom event, and I'm not using e.code anymore.